python/requirements-dev.txt

```diff
@@ -1,6 +1,7 @@
+-f https://dist.apache.org/repos/dist/dev/spark/v3.0.0-rc1-bin/
```

Comment:
why find-links from here?

1. it is not the same spark version as we're installing
2. pip will bring pyspark dependencies already

Reply:
I'm also inheriting this from the original spark-3.0.0 branch. I don't mind dropping this unless @mengxr had a good reason.

Reply:
I can at least set it to spark 3.1.1.